blocks/measuring-impact-cards/measuring-impact-cards.css

```diff
@@ -0,0 +1,134 @@
+.measuring-impact-cards{
+    display: flex;
+    flex-flow: row wrap;
+    padding: 0;
+    gap: 20px;
+}
+
+.impact-card-wrapper{
```

Comment:
Can you please namespace all the selectors within block class name?

Reply:
Done